source/pcic/client.py

```diff
@@ -26,11 +26,11 @@ def __init__(self, address, port, autoconnect=True, timeout=5):
         self.debug = False
         self.debugFull = False
 
-    def __enter__(self):
-        return self
-
-    def __exit__(self, exc_type, exc_val, exc_tb):
-        self.disconnect()
+    # def __enter__(self):
+    #     return self
+    #
+    # def __exit__(self, exc_type, exc_val, exc_tb):
+    #     self.disconnect()
```

Comment:
Is this change intentional? If yes, then better remove the lines instead of commenting them out.

Reply:
Fixed in commit 231f3fc9e5c9a4bf55caddcc65ac70e49a27ace4

Reply:
Looks good to me, PR is ready for merge I would say.